buckw

```diff
@@ -69,16 +69,15 @@ removeWatch ( ) {
 getToClean ( ) {
     ensureWatch
 
-    watchman --output-encoding=json -j 2>&1 <<-EOT
+    watchman --output-encoding=json --no-pretty -j 2>&1 <<-EOT
 ["query", "$WORKING_DIR", {
     "expression": ["allof",
         ["type", "f"],
         ["anyof",
-            ["imatch", "**/BUCK", "wholename"],
-            ["imatch", ".buckconfig.local", "wholename"]
+            ["name", ["BUCK", ".buckconfig.local"]]
```

Comment:
@wez I have fixed this up as you suggested. Thanks!

Learnt a lot more about watchman through this exercise :)